.github/actions/sign-attestations/src/main.ts

```diff
@@ -17,11 +17,6 @@ import { sigstore } from "sigstore";
 import * as path from "path";
 import * as tscommon from "tscommon";
 
-const signOptions = {
```

Comment:
Are these now default params in v1.9?

Reply:
In v1.9, it defaults to CI provider when no other strategy has been configured, instead of defaulting as the first strategy like before in v1.8. Removing signOptions, lets it fallback to CI provider to get OIDC token for GHA.

You can see that change here: https://github.com/sigstore/sigstore-js/blob/sigstore%401.9.0/packages/client/src/config.ts#L130-L139